performance/font-loading.md

```diff
@@ -0,0 +1,60 @@
+# Font Loading
+
+## When to use webfonts
+Before using webfonts it's worth understanding what benefits you believe a webfont will provide your product and assess those against the potential impact they may have on things like accessibility or page speed. For example:
+
+- System fonts are battle tested and were produced with legibility in mind. Using webfonts may reduce the readability of the site, especially for people with dyslexia.
```

Comment:
```suggestion
- System fonts are extensively tested and produced with legibility in mind. Using webfonts may reduce the readability of the site, especially for people with dyslexia.
```
Want to avoid militaristic language